src/hb-aat-feat-table.hh

```diff
@@ -32,13 +32,28 @@
 
 namespace AAT {
 
-struct FeatureName {
+struct SettingName
+{
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
 
+  protected:
+  HBUINT16	setting;	/* The setting. */
+  NameID	nameIndex;	/* The name table index for the setting's name. */
+  public:
+  DEFINE_SIZE_STATIC (4);
+};
+
+struct FeatureName
+{
   inline bool sanitize (hb_sanitize_context_t *c, const void *base) const
   {
     TRACE_SANITIZE (this);
     return_trace (c->check_struct (this) &&
-      (base+settingTable).sanitize (c, nSettings, base));
+		  (base+settingTable).sanitize (c, nSettings));
```

Comment:
This is changed from the original code. Previous code was passing just to make sure the inner sanitization.